modules/cudf/src/column.ts

```diff
@@ -1060,6 +1060,8 @@ export interface Column<T extends DataType = any> {
    * @returns column without duplicate values
    */
   drop_duplicates(nullsEqual?: boolean, memoryResource?: MemoryResource): Column<T>;
+
+  getJSONObject(jsonPath?: string, memoryResource?: MemoryResource): Column<T>;
```

Comment:
Should `jsonPath` be optional? What does the function do with no path provided?

Reply:
python function: https://github.com/rapidsai/cudf/blob/8ae73d50cee62a282cba152cbce7d23449b5a179/python/cudf/cudf/core/column/string.py#L2184

where json_path is mandatory

Reply:
wouldn't a compile time error be preferable to a runtime exception? I guess the point is somewhat moot since users won't really be interacting with `Column` methods directly themselves/